arrow-fx-coroutines/src/main/kotlin/arrow/fx/coroutines/Enviroment.kt

```diff
@@ -98,3 +100,15 @@ internal class DefaultEnvironment(override val ctx: CoroutineContext) : Environm
       res.fold(a, e) // Return error to caller
     })
 }
+
+private fun CoroutineContext.asDefaultCtx(): CoroutineContext = plus(DefaultContext(this))
+
+internal class DefaultContext(val ctx: CoroutineContext) : AbstractCoroutineContextElement(DefaultContext) {
+  companion object Key : CoroutineContext.Key<DefaultContext>
+}
+
+internal inline fun CoroutineContext.defaultContext(ctx: CoroutineContext): CoroutineContext =
+  this[DefaultContext] ?: ctx
```

Comment:
```suggestion
  this[DefaultContext]?.ctx ?: ctx
```

Need to unwrap from `DefaultContext`.

Reply:
I had this fixed in my test context branch and completely forgot to put this here, thanks! :see_no_evil: